packages/ripple-nuxt-tide/lib/pages/Tide.vue

```diff
@@ -53,7 +53,7 @@
 
       <template slot="belowContent">
         <component class="rpl-site-constrain--on-all" v-if="campaignSecondary" :is="campaignSecondary.component" v-bind="campaignSecondary.data"></component>
-        <component :is="contentRating" class="rpl-site-constrain--on-all"></component>
+        <component :is="contentRating" :siteSectionName="siteSectionName" class="rpl-site-constrain--on-all"></component>
```

Comment:
I guess below will work without need for `siteSectionName` method.
```suggestion
        <component :is="contentRating" :siteSectionName="page.appSection?.name" class="rpl-site-constrain--on-all"></component>
```
That's called `optional chaining operator` is a new way to dealing with undefined object properties.